.devops/repo/src/commands/build.rs

```diff
@@ -0,0 +1,21 @@
+use crate::utils::{execute, print_applications};
+use std::process::Command;
+
+pub(crate) fn run(apps: Vec<String>, root: String) {
+    let mut errors: Vec<String> = vec![];
+
+    for app in apps {
+        println!("\nBuilding {}", app);
+
+        // TODO: safety
+        let project = std::fs::canonicalize(format!("{}/{}/project", root, app)).unwrap();
+
+        execute(
+            Command::new("forc").current_dir(project).arg("build"),
+            &mut errors,
+            &app,
+        );
+    }
+
+    print_applications(errors, "Errors found in".to_string());
```

Comment:
It could be worth creating a const for the error messages that's passed into `print_applications`, i.e. 

`const ERRORS_FOUND_MESSAGE: &str = "Errors found in";`